src/main/java/mate/controller/driver/AddDriverController.java

```diff
@@ -23,10 +25,11 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
     @Override
     protected void doPost(HttpServletRequest req, HttpServletResponse resp)
             throws IOException, ServletException {
-        String name = req.getParameter("name");
-        String licenceNumber = req.getParameter("licence_number");
-        Driver driver = new Driver(name, licenceNumber);
+        Driver driver = new Driver(req.getParameter("name"),
+                req.getParameter("licence_number"),
+                req.getParameter("login"),
+                req.getParameter("password"));
         driverService.create(driver);
-        resp.sendRedirect("/drivers/add");
+        resp.sendRedirect("/login");
```

Comment:
Why do you redirect to login controller here?

Reply:
still don't understand why do you redirect to login page after adding new driver?